src/utils/clmfates_interfaceMod.F90

```diff
@@ -305,6 +305,7 @@ subroutine CLMFatesGlobals1(surf_numpft,surf_numcft,maxsoil_patches)
      integer                                        :: pass_nocomp
      integer                                        :: pass_sp
      integer                                        :: pass_masterproc
+     integer                                        :: pass_luh
```

Comment:
Could the existing `pass_use_luh` not be used for consistency in terms of naming (i.e., with `pass_use_potentialveg`)? If not, `pass_use_luh` seems to not be used anymore, so it should be deleted.

Reply:
I previously removed the "use" just to be less  verbose, nothing more. I'm fine with adding it back in.
@ekluzek, the pass_use_potentialveg is a separate flag from pass_use_luh, I think @samsrabin was pointing it out as an example of convention. 
I'm happy to expand this PR though if you would like to update the name of that flag to something less "overlappy" with native ctsm.  @ckoven, could you weigh in here on potential alternative names for [use_fates_potentialveg](https://github.com/ESCOMP/CTSM/blob/master/src/main/controlMod.F90#L241).

Reply:
Hi All -- My preference is to use "Potential Veg" or similar, rather than "Spinup". The reason is that potential vegetation is a more accurate description of what the relevant code does. Spinup is more of a workflow concept; a user might use potential vegetation apart from spinup, and might do spinup without using potential vegetation mode. It also might make it sound to a user like potential veg has something to do with AD spinup, which it doesn't.  In terms of the conceptual overlap of how the phrase is used in non-FATES CTSM, I think that it makes sense that both vegetation models might use the same vocabulary for something that is conceptually similar?  I guess I don't see why that is a problem. It seems sort of analogous to how both models have stomatal conductance switches that share names. But I guess another possibility would be to call it something like "All Primary Land Mode"? But I feel like that is more verbose.